cookbook/import_export/create-connector.rst

```diff
@@ -42,6 +42,10 @@ The writer is implemented in the class ``Pim\Component\Connector\Writer\DummyIte
 
 We'll explain in next cookbook chapters how to create your own elements with real logic inside.
 
+.. warning::
+
+   It is also possible to create multiple job configuration files by placing them in a folder ``Resources/config/batch_jobs``.
```

Comment:
`Resources/config/batch_jobs/any_name.yml` or `Resources/config/batch_jobs/xxx.yml` could be more understandable I think.